api/services/administration/general/paths/healthCheckID.yaml

```diff
@@ -69,7 +69,7 @@ delete:
     - "Health Check Service"
   parameters:
     - $ref: "../../../../commons/parameters/accountId.yaml"
-    - in: "query"
+    - in: "path"
```

Comment:
This is more than a syntactic change, did you check with our API source-code that these are the correct parameters?

Reply:
The parameter was added as a query parameter by mistake since it appears in the path provided for the route in the documentation as a path parameter. Also verified in API source code.